### ISSUE TITLE
Clarification on setAriaDescribedBy

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#178]
I saw:
```js
accessibleFormNode.setAriaDescribedBy( accessibleDescription.id );
```
where accessibleDescription is a AccessibleDescriptionNode, so its ID is presumably the Node's auto-generated ID.

This this how things are set? Also Node.id is a number, setAriaDescribedBy says it takes a string.

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
```setAriaDescribedBy``` from AccessibleNode has been replaced with ```setAriaDescribedByElement()``` in Accessibility.js.

I have changed the above line and added documentation such as:
```js
    // set the description relation between the form and this node's description element
    // when focus enters the form, the content under the description element should be read by the screen reader
    // see setAriaDescribedByElement() in Accessibility.js mixin for more information
    accessibleFormNode.setAriaDescribedByElement( this.descriptionElement );
```

Documentation in Accessibility.js:
```js
/**
 * Set the 'aria-describedby' element for this node's DOM element. The value of the 'aria-describedby'
 * attribute is a string id that references the desired HTML element.  Upon focus, a screen reader should
 * read the content under the HTML element referenced by the 'aria-describedby' id. Exact behavior will depend
 * slightly on user agent.
 * @public
 *
 * @param {HTMLElement} descriptionElement
 */
```

@jonathanolson is this sufficient to clarify things?

**Comment by jonathanolson** [comment]
Yup, looks good. Thanks!
